app/views/thredded/posts_common/_actions.html.erb

```diff
@@ -1,11 +1,24 @@
-<% if post.can_update? %>
-  <%= link_to t('thredded.posts.edit'), post.edit_path, class: 'thredded--post--edit' %>
-<% end %>
+<div class='thredded--post--dropdown'>
```

Comment:
This currently displays the dropdown toggle even if there are no actions available to the user.

I can think of two extension-friendly ways of fixing this:

1. With CSS: Move the toggle outside and after the actions container, and set:
    ```scss
    .thredded--post--dropdown--actions:empty + .thredded--post--dropdown--toggle {
      display: none;
    }
    ```
2. With HTML: render the actions to a string via `capture` first, and then check that the result is not blank before rendering the dropdown.

Either one is fine by me